Launcher/config.json

```diff
@@ -6,10 +6,10 @@
   // two predefined environments, 'backtesting' and 'live', feel free
   // to add more!
 
-  "environment": "backtesting",
+  "environment": "live-paper",// "live-paper", "backtesting", "live-interactive", "live-interactive-iqfeed"
 
   // algorithm class selector
-  "algorithm-type-name": "BasicTemplateAlgorithm",
+  "algorithm-type-name": "BasicTemplateOptionsHistoryAlgorithm",
```

Comment:
Good point. I updated config.json for those using/testing futures branch locally. Now that we merge it I am not sure we need this config.json anymore.